src/modules/fetchlike.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] Kindly enable eslint for it is not allowed to disable linters. linters help us write clean code therefore it is against microverse policy to disable linters.